x-pack/plugin/transform/qa/multi-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/transform/integration/continuous/LatestContinuousIT.java

```diff
@@ -38,11 +38,17 @@
 import static org.hamcrest.Matchers.is;
 import static org.hamcrest.Matchers.not;
 
-@AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/68394")
 public class LatestContinuousIT extends ContinuousTestCase {
 
     private static final String NAME = "continuous-latest-test";
 
+    private static final Map<String, Object> RUNTIME_MAPPINGS =
+        new HashMap<>() {{
```

Comment:
I think this uses java features that we can't use in `7.x`. Unless I'm wrong, could we change this to the older (and unfortunately more verbose) syntax to avoid conflicts on future backports?

Reply:
Done.

I usually add the generic parameters during backport, but can do it here as well.